TouchSky/TouchSky/TouchSky/Views/AllList.swift

```diff
@@ -1,33 +1,33 @@
 //
-//  PeakList.swift
+//  AllList.swift
 //  TouchSky
 //
 //  Created by Alexa Nicole Hugenberg on 9/19/22.
 //
 
 import SwiftUI
 
-struct PeakList: View {
+struct AllList: View {
```

Comment:
I actually really like the name `PeakList` compared to `AllList`, and I think a nice improvement that you can make is to get rid of all of the separate views for `AllList`, `CompletedList`, and `ToDoList`, and drive them all from a single view implementation that adjusts depending on the data that needs to be displayed.

I'm not sure exactly the best way to explain this but I'll start here and go through the code leaving comments about that as I get to relevant spots.